README.md

```diff
@@ -1,20 +1,24 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
-
-(b) Where in your JS file should you put it, and WHY? 
+(a) What is a click handler?
+  A click handler is a function that calls the 'click' method and then will say what should happen when a user clicks the click handler.
+(b) Where in your JS file should you put it, and WHY?
+  Click handlers should be placed in the $document ready function at the top of the file.
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+  If $ is undefined jQuery is probably not included in the file and will need to be added in by including a link in the html file
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+  All sorts of fun stuff! Basically everything that you want to show up when the page is loaded.
 
 ### Question 4
-In an AJAX GET request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? 
+In an AJAX GET request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent?
+  The argument of done is a function that will call up the "success" message in the console after the GET request has been made and the data has been loaded.
```

Comment:
You are technically correct, though I was asking more specifically about `data` - which is whatever the server returns to you (for this API, a set of all users).